### PR TITLE
Add option to tunneld for authenticating clients with certificates.

### PR DESCRIPTION
**Pull request by goriccardo**

### CODE REVIEW

File: cmd/tunneld/options.go [hunk 1]
```diff
@@ -36,15 +36,16 @@ func init() {
...
-	clients    string
-	logTo      string
-	logLevel   int
-	version    bool
+	httpAddr      string
+	httpsAddr     string
+	tunnelAddr    string
+	tlsCrt        string
+	tlsKey        string
+	clientsRootCA string
```
Comment (mmatczuk) [inline cmd/tunneld/options.go]: can w make it just rootCA

**Comment by mmatczuk** [comment]
Otherwise it looks good and possibly solves #43

**Comment by goriccardo** [comment]
> Otherwise it looks good and possibly solves #43

Sorry @mmatczuk, apparently I can't find your code review.

File: cmd/tunneld/options.go (hunk 1 above)
Comment (mmatczuk) [reply cmd/tunneld/options.go]: similar for flag and so on

Comment (goriccardo) [reply cmd/tunneld/options.go]: Done 👍

**Comment by mmatczuk** [comment]
Awesome, thanks!

**Pull request merged**